cmake/aditof-config.cmake.in

```diff
@@ -1,14 +1,15 @@
 get_filename_component(aditof_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 
 set(aditof_INCLUDE_DIRS "@CONFIG_INCLUDE_DIRS@")
+set(WITH_NETWORK "@WITH_NETWORK@")
 
 if (NOT glog_FOUND)
         find_package(glog 0.3.5 REQUIRED)
 endif()
-if (NOT Protobuf_FOUND)
+if (WITH_NETWORK AND NOT Protobuf_FOUND)
```

Comment:
It should be more readable to group Protobuf_FOUND and Libwebsockets_FOUND under a single if (WITH_NETWORK)

Reply:
fixed